src/components/ChatEntry.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({ id, sender, body, timeStamp, changeLiked, liked }) => {
+  const side = sender === 'Vladimir' ? 'local' : 'remote';
```

Comment:
The ternaries are great for situations like this right? They are the perfect tool to use when wanting conditionally apply styling!